DuckDuckGo/Preferences/Model/VPNPreferencesModel.swift

```diff
@@ -91,27 +91,24 @@ final class VPNPreferencesModel: ObservableObject {
     private let vpnXPCClient: VPNControllerXPCClient
     private let settings: VPNSettings
     private let pinningManager: PinningManager
-    private let internalUserDecider: InternalUserDecider
     private var cancellables = Set<AnyCancellable>()
 
     init(vpnXPCClient: VPNControllerXPCClient = .shared,
          settings: VPNSettings = .init(defaults: .netP),
          pinningManager: PinningManager = LocalPinningManager.shared,
-         defaults: UserDefaults = .netP,
-         internalUserDecider: InternalUserDecider = NSApp.delegateTyped.internalUserDecider) {
```

Comment:
Also removing the internal user limitation for the feature, and the associated feature flag.

Reply:
In the future we could likely use an internal feature flag, and then just flip it to live when we're ready. (That would also allow for rolling it out incrementally)